### PR TITLE
Select deployment to schedule job

### PR DESCRIPTION
**Pull request by satr**

### CODE REVIEW

File: src/components/component/scheduled-job/use-get-deployments.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  appName: string,
+  envName: string,
+  jobComponentName: string
+): AsyncLoadingResult<Array<Readonly<DeploymentItemModel>>> {
+  const encAppName = encodeURIComponent(appName);
+  const encEnvName = encodeURIComponent(envName);
+  const encJobComponentName = encodeURIComponent(jobComponentName);
+  console.log(
+    `useGetDeployments: appName=${encAppName}, envName=${encEnvName}, jobComponentName=${encJobComponentName}`
+  );
```
Comment (Pespiri) [inline src/components/component/scheduled-job/use-get-deployments.ts:18]: remove debug logging
```suggestion

```

File: src/components/component/scheduled-job/scheduled-batch-list.tsx [hunk 2]
```diff
@@ -19,33 +19,35 @@ import { connect } from 'react-redux';
...
+import { errorToast } from '../../global-top-nav/styled-toaster';
+import { ProgressStatusBadge } from '../../status-badges';
+import { Duration } from '../../time/duration';
+import { RelativeToNow } from '../../time/relative-to-now';
 import { JobContextMenu } from './job-context-menu';
 import { JobDeploymentLink } from './job-deployment-link';
 
 import './style.css';
+import { ScrimPopup } from '../../scrim-popup';
+import { RestartBatch } from './restart-batch';
```
Comment (Pespiri) [inline src/components/component/scheduled-job/scheduled-batch-list.tsx:50]: sort imports

File: src/components/component/scheduled-job/scheduled-job-list.tsx [hunk 3]
```diff
@@ -20,37 +21,38 @@ import { connect } from 'react-redux';
...
+import { Duration } from '../../time/duration';
+import { RelativeToNow } from '../../time/relative-to-now';
 import { JobContextMenu } from './job-context-menu';
 import { JobDeploymentLink } from './job-deployment-link';
-import { Payload } from './scheduled-job/payload';
+import { Payload } from './payload';
 
-import './style.css';
+import '../style.css';
+import { RestartJob } from './restart-job';
```
Comment (Pespiri) [inline src/components/component/scheduled-job/scheduled-job-list.tsx:55]: sort imports

**Review by Pespiri — CHANGES_REQUESTED** [review]

**Review by Pespiri — APPROVED** [review]

File: src/components/component/scheduled-job/scheduled-batch-list.tsx (hunk 2 above)
Comment (satr) [reply src/components/component/scheduled-job/scheduled-batch-list.tsx:48]: Fixed

**Pull request merged**